homeassistant/components/uptimerobot/__init__.py

```diff
@@ -51,64 +46,3 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
-
-
-class UptimeRobotDataUpdateCoordinator(DataUpdateCoordinator[list[UptimeRobotMonitor]]):
```

Comment:
The platforms are still importing from the root `from . import UptimeRobotDataUpdateCoordinator` that should change to `from .coordinator import UptimeRobotDataUpdateCoordinator`

Reply:
Thx, this fixed now.